### PR TITLE
Test Federal Senate integration in CI

### PR DESCRIPTION
**Pull request by Irio**
Chamber of Deputies still doesn't work (given its higher memory usage), but will do after some effort.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
+python: 3.6
+cache: pip
+install:
   - ./setup
   - pip install coveralls
-script: 
+script:
   - coverage run rosie.py test
-
+  - python rosie.py run federal_senate
```
Comment (cuducos) [inline .travis.yml]: Are we actually using `run` not `test` for… testing?

Comment (Irio) [inline .travis.yml]: Yes, that's the idea. In the line above we run all the written tests. Here, we `run federal_senate`. If doesn't work for any reason, will return a non-zero exit code and make the suite fail on CI.

Comment (cuducos) [inline .travis.yml]: Why? Because we don't have integrated tests (a journey) for the Senate app?

Comment (Irio) [inline .travis.yml]: Yes.

Comment (jtemporal) [inline .travis.yml]: so this is basically a workaround to not having journey tests? and if so wouldn't it be wiser to not do this and open up an issue "Create journey tests for federal senate"?

Comment (cuducos) [inline .travis.yml]: I do agree, @jtemporal — but I also see the point: better to run and make sure it doesn't crash than have nothing in the CI at all. Maybe both?

Comment (jtemporal) [inline .travis.yml]: both sounds good! @cuducos

**Comment by jtemporal** [comment]
So I'm merging this and opening an issue based on [this comment by @cuducos on having journey tests](https://github.com/datasciencebr/rosie/pull/59#discussion_r125341315)

**Pull request merged**